src/utils/d2c_messaging/tests/d2c_messaging_ut.cpp

```diff
@@ -283,14 +283,14 @@ static void OnMessageProcessCompleted_SaveStatus_And_Signal(void* context, ADUC_
     pthread_cond_signal(&g_d2cMessageProcessedCond);
 };
 
-// Make sure that we can uninitialize cleanly while there's a message in-progress.
-TEST_CASE("Uninitialization - in progess message")
+// Make sure that we can deinitialize cleanly while there's a message in-progress.
+TEST_CASE("Deinitialization - in progress message", "[.][functional]")
```

Comment:
i like the use of the tags here. (TIL `[.]` indicates that it's not included by default